package.json

```diff
@@ -1,66 +1,79 @@
 {
   "private": true,
-  "license": "MIT",
+  "engines": {
+    "node": ">=6",
+    "npm": ">=3"
+  },
   "scripts": {
-    "install": "run-s -s install:*",
-    "install:mkdirp-bin": "lerna exec -- mkdirp ./node_modules/.bin/",
-    "install:bootstrap": "lerna bootstrap",
-    "install:symlink-bin": "lerna exec -- node ../../scripts/symlink-bin.js",
-    "clean": "lerna exec -- rm -rf lib/* && lerna clean --yes",
-    "lint": "lerna exec -- npm run lint -s",
-    "build": "lerna exec -- npm run build -s",
-    "test": "lerna exec -- npm run test -s",
-    "coverage": "lerna exec -- npm run coverage -s",
-    "codecov": "lerna exec -- npm run codecov -s",
-    "commit": "git-cz || true",
-    "check-updates": "run-p -s check-updates:*",
-    "check-updates:root": "ncu --error-level 2",
-    "check-updates:packages": "lerna exec -- ncu --error-level 2",
```

Comment:
What I tried to say is that the control to run them individually is removed, not that it's completely removed, but okay. As for upgrading if we remove that we'd need to do it manually or what is the alternative you propose? I was thinking it's a nice script to ease keeping everything up-to-date (so not something CI would run but for all maintainers to use).